param/_utils.py

```diff
@@ -203,15 +203,6 @@ def _is_mutable_container(value):
     return issubclass(type(value), MUTABLE_TYPES)
```

Comment:
Unrelated but this looks more correct to me
```suggestion
    return isinstance(value, MUTABLE_TYPES)
```